modules/apps/product-navigation/product-navigation-control-panel/src/main/java/com/liferay/product/navigation/control/panel/internal/application/list/UsersPanelCategory.java

```diff
@@ -21,7 +21,7 @@
 @Component(
 	property = {
 		"panel.category.key=" + PanelCategoryKeys.CONTROL_PANEL,
-		"panel.category.order:Integer=100"
+		"panel.category.order:Integer=0"
```

Comment:
We shouldn't be setting this to `0`. If we do that we make it difficult for customers to customize this and add entries before ours.

Reply:
Ah I see, I'll change this.